src/charm.py

```diff
@@ -51,7 +51,7 @@ def __init__(self, *args):
             "metrics-endpoint",
             jobs=[
                 {
-                    "job_name": self.unit.name,
+                    "job_name": self.model.app.name,
```

Comment:
This is app name, but their "temporal stability" is probably comparable.
However, redeploying under a different app/model name would be reflected in labels, which would break timeseries continuity anyway.
Regardless, for avalanche it doesn't matter. In fact, maybe for avalanche it's better to break continuity.

Reply:
That is reasonable. But shouldn't you document that behaviour in that case ?

Reply:
> redeploying under a different app/model name would be reflected in labels, which would break timeseries continuity anyway

This is a prometheus feature, not avalanche side effect: if the app/model name changes and everything else kept the same, it will break timeseries continuity.

Reply:
Avalanche is choosing to set a job name that can change. This is not a Prometheus feature. What needs to be documented is that changing Avalanche's model or app name will result in a discontinuity in the metric.

Reply:
Correct, but that will happen regardless of the job name, iiuc:
A different app name or model name would be reflected in metric's labels, which would discontinue the timeseries regardless of job name.